test/hotspot/gtest/utilities/test_globalDefinitions.cpp

```diff
@@ -295,3 +295,30 @@ TEST(globalDefinitions, format_specifiers) {
   // Check all platforms print this compatibly without leading 0x.
   check_format(UINT64_FORMAT_0,        (u8)0x123,         "0000000000000123");
 }
+
+TEST(globalDefinitions, jlong_from) {
+  jlong val = jlong_from(0xFF, 0);
+  EXPECT_EQ(val, 0x00000000FF00000000);
```

Comment:
Should be using `CONST64` here and elsewhere.

Reply:
Done. 
All windows tests had been passed.